static/styles/settings.css

```diff
@@ -1540,15 +1545,16 @@ input[type="checkbox"] {
 
     .content-wrapper {
         position: absolute;
-        left: 251px;
-        /* the width of the settings sidebar this is right of is 250px + 1px border. */
-        width: calc(100% - 250px - 1px);
+        left: 250px;
+        /* the width of the settings sidebar is 250px */
+        width: calc(100% - 250px);
         height: 100%;
         overflow: hidden;
 
         .settings-header {
             width: 100%;
-            height: 43px;
+            height: 45px;
+            box-sizing: border-box;
```

Comment:
Used CSS Box Model to make the height predictable, i.e. 45px which was the computed height before using CSS Box Model. Also this matches with the height of the tab switcher. No changes will be observed visually but this makes the code more consistent.